package.json

```diff
@@ -85,7 +87,8 @@
     "svgo": "^2.8.0",
     "ts-jest": "^28.0.7",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.0-beta"
+    "typescript": "^4.8.0-beta",
+    "vitest": "^0.21.1"
```

Comment:
also, install `c8` (for test coverage), `@vitest/ui`, and `vite` (required for single-file debugging in VSCode I don't know why.)

Remove jest config and jest packages.